lib/index.js

```diff
@@ -44,5 +44,8 @@ History.prototype.lengthForward = require('./prototype/length-forward')
 History.prototype.backward = require('./prototype/backward')
 History.prototype.forward = require('./prototype/forward')
 History.prototype.move = require('./prototype/move')
+History.prototype.forgetAllInDirection = require('./prototype/forget-all-in-direction')
```

Comment:
This doesn't seem like a useful external API. Why would someone prefer passing in a magic string over calling the appropriate method directly?